neon_llm_chatgpt/chatgpt.py

```diff
@@ -40,21 +40,33 @@ class ChatGPT(NeonLLM):
 
     def __init__(self, config):
         super().__init__(config)
-        self.model = config["model"]
+        self.model_name = config["model"]
         self.role = config["role"]
         self.context_depth = config["context_depth"]
         self.max_tokens = config["max_tokens"]
         self.api_key = config["key"]
         self.num_parallel_processes = config["num_parallel_processes"]
         self.warmup()
 
+    @property
+    def tokenizer(self) -> None:
+        return self._tokenizer
+
+    @property
+    def tokenizer_model_name(self) -> str:
+        return ""
+
```

Comment:
Are these properties used anywhere or just leftover changes?

Reply:
@NeonDaniel intentionally, or else there will be an error, because it is defined in parent abstract class
But chatgpt handles tokenization internally